src/elements/core/styles/mixins/lists.scss

```
@@ -76,28 +76,119 @@
-  --sbb-step-list-padding-inline: var(--sbb-spacing-responsive-xxs);
-  --sbb-step-list-marker-to-text-gap: var(--sbb-spacing-responsive-xxxs);
-  --sbb-step-list-border-radius: var(--sbb-border-radius-4x);
+@mixin description-list {
+  // Support both top level usage (& is empty / falsy)
+  // and applied to a selector usage.
+  #{if(&, '&:where(dl)', 'dl')} {
+    @include typo.text-s--regular;
+
+    margin: 0;
+    padding: 0;
+    display: grid;
+
+    // Ensure that description is always attached to the label
+    grid-template-columns: auto minmax(20%, 1fr);
+    gap: var(--sbb-spacing-fixed-1x) var(--sbb-spacing-fixed-2x);
+    color: var(--sbb-color-iron);
+
+    :is(dt, dd) {
+      margin: 0;
+      padding: 0;
+    }
+  }
+}
+
+@mixin base-marker-list {
+  --sbb-list-marker-dimensions: #{functions.px-to-rem-build(34)};
+  --sbb-list-marker-padding-inline: var(--sbb-spacing-responsive-xxs);
+  --sbb-list-marker-to-text-gap: var(--sbb-spacing-responsive-xxxs);
 
   // Additional space from overall li padding to the text because text
   // has to be centered to marker number.
-  --sbb-step-list-text-to-marker-block-offset: calc(
+  --sbb-list-text-to-marker-block-offset: calc(
     0.5 *
       (
-        var(--sbb-step-list-marker-dimensions) - var(--sbb-typo-line-height-body-text) *
+        var(--sbb-list-marker-dimensions) - var(--sbb-typo-line-height-body-text) *
           var(--sbb-text-font-size)
       )
   );
-  --sbb-step-list-vertical-gap: var(--sbb-spacing-fixed-1x);
+
+  &:where(.sbb-text-xl) {
+    @include mediaqueries.mq($from: medium) {
+      --sbb-list-marker-dimensions: #{functions.px-to-rem-build(41)};
+    }
+  }
 
   list-style: none;
   margin: 0;
   padding: 0;
-  counter-reset: steps;
+
+  /* stylelint-disable-next-line no-descending-specificity */
+  > li {
+    position: relative;
+    padding-inline: calc(
+        var(--sbb-list-marker-padding-inline) + var(--sbb-list-marker-dimensions) +
+          var(--sbb-list-marker-to-text-gap)
+      )
+      var(--sbb-list-marker-padding-inline);
+    min-height: var(--sbb-list-marker-dimensions);
+
+    &::before {
+      @include typo.text-xxs--bold;
+
+      position: absolute;
+      height: var(--sbb-list-marker-dimensions);
+      width: var(--sbb-list-marker-dimensions);
+      margin-block-start: calc(-1 * var(--sbb-list-text-to-marker-block-offset));
+      inset-inline-start: var(--sbb-list-marker-padding-inline);
+      border-radius: 50%;
+    }
+
+    + li {
+      margin-block-start: var(--sbb-list-marker-vertical-gap);
+    }
+
+    p {
+      margin-block: 0;
+    }
+
+    // P has to be redefined to be more specific
+    > * + p {
+      margin-block-start: var(--sbb-spacing-responsive-xxxs);
+    }
+  }
+}
+
+@mixin checkup-list {
+  @include base-marker-list;
+
+  --sbb-checkup-list-marker-icon: url('https://icons.app.sbb.ch/icons/circle-tick-medium.svg');
+  --sbb-checkup-list-marker-icon-color: var(--sbb-color-black);
```

Would `current` or `inherit` work? If yes, we could leave out --sbb-checkup-list-color var as the consumer could just set "color:" on the list itself (I did not check if it really works ;) )
```suggestion
  --sbb-checkup-list-marker-icon-color: current;
```

---

current and inherit don't seem to work, but the keyword currentColor works like you described